eng/Version.Details.xml

```diff
@@ -19,5 +19,9 @@
       <Uri>https://github.com/NuGet/NuGet.Client</Uri>
       <Sha>c1f6915918b82c096bbd666bd9c18528f1f70630</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.Helix.Sdk" Version="5.0.0-beta.19524.2">
+      <Uri>https://github.com/dotnet/arcade</Uri>
+      <Sha />
```

Comment:
```suggestion
      <Sha>0e9ffd6464aff37aef2dc41dc2162d258f266e32</Sha>
```

Reply:
Same as Arcade SDK.

Reply:
Can you move this up next to Arcade and SignTool also.

Reply:
HostModel and NuGet should be product, not toolset dependencies, but that`s for another change.